src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCost = 40;
+  const totalCost = days * dailyRentalCost;
+  const maxDiscountDays = 7;
+  const minDiscountDays = 3;
+  const smallDiscount = 20;
+  const largeDiscount = 50;
+
+  if (days >= minDiscountDays && days < maxDiscountDays) {
```

Comment:
Consider using an `else if` statement instead of a separate `if` statement for better readability and logical flow.

Reply:
Fixed.